internal/host/validator.go

```diff
@@ -1626,3 +1635,73 @@ func (v *validator) noSkipMissingDisk(c *validationContext) (ValidationStatus, s
 	}
 	return ValidationSuccess, successMessage
 }
+
+func (v *validator) serviceCanConnectToSpokeKubeAPI(c *validationContext) (ValidationStatus, string) {
```

Comment:
What is the performance of this validation? 
Imagine managing 2K day2 clusters

Reply:
Actually it's a validation that  will run only before we start day2 installation so maybe it's fine. 
I confused it with a validation on the cluster and not on the host.